packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -231,22 +231,21 @@ class DatePicker extends React.Component {
     // Modern browsers support event.relatedTarget but event.relatedTarget returns null in IE 10 / IE 11.
     // IE 11 sets document.activeElement to the next focused element before the blur event is called.
     const activeTarget = event.relatedTarget ? event.relatedTarget : document.activeElement;
-
     // Handle blur only if focus has moved out of the entire date picker component.
     if (!this.datePickerContainer.current.contains(activeTarget)) {
       if (this.props.onBlur) {
         const format = DateUtil.getFormatByLocale(this.props.intl.locale);
-        const isCompleteDate = DateUtil.isValidDate(this.dateValue, format);
-        const iSOString = isCompleteDate ? DateUtil.convertToISO8601(this.dateValue, format) : '';
+        const isCompleteDate = DateUtil.isValidDate(this.state.selectedDate, format);
+        const iSOString = isCompleteDate ? DateUtil.convertToISO8601(this.state.selectedDate, format) : '';
         let isValidDate = false;
 
-        if (this.dateValue === '' || (isCompleteDate && this.isDateWithinRange(DateUtil.createSafeDate(iSOString)))) {
+        if ((isCompleteDate && this.isDateWithinRange(DateUtil.createSafeDate(iSOString)))) {
           isValidDate = true;
         }
 
         const options = {
           iSO: iSOString,
-          inputValue: this.dateValue,
+          inputValue: iSOString,
```

Comment:
What's the reason for changing this to be the ISO format, which is a duplicate value returned in the `iSO` key? This used to be what ever value the user entered. This would break consumers since it will no longer be what they expect.

Reply:
Changed back! [452f10b](https://github.com/cerner/terra-framework/pull/1004/commits/452f10bd2fac12ea3c50aff9d7b90d0cff1d3b01)